domainlab/utils/generate_benchmark_plots.py

```diff
@@ -19,40 +32,77 @@ def gen_benchmark_plots(agg_results: str, output_dir: str):
 
     agg_results: path to the csv file
     output_dir: path to a folder which shall contain the results
+    skip_gen: Skips the actual plotting, used to speed up testing.
```

Comment:
I don't see skip_gen as an argument to this function, I also don't remember that I have add something like this. @smilesun did you to add this feature?

Reply:
Yes, this was me.
I still think we can close this pr